### PR TITLE
Improve `grade-school` description

### PR DESCRIPTION
**Pull request by Elahi-cs**
This PR follows [this discussion on the forum](https://forum.exercism.org/t/improve-grade-school-description/13694). A complete list of the changes and their rationale can be seen there.

### CODE REVIEW

File: exercises/grade-school/description.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-  - "Let me think.
-    We have Anna, Barb, and Charlie in grade 1, Alex, Peter, and Zoe in grade 2 and Jim in grade 5.
-    So the answer is: Anna, Barb, Charlie, Alex, Peter, Zoe and Jim"
+  Grades should be sorted as 1, 2, 3, etc., and students within a grade should be sorted alphabetically by name.
+  - "Who is enrolled in school right now?"
+  - "Let me think. We have Anna, Barb, and Charlie in grade 1, Alex, Peter, and Zoe in grade 2, and Jim in grade 5. So the answer is: Anna, Barb, Charlie, Alex, Peter, Zoe, and Jim."
 
-Note that all our students only have one name (It's a small town, what do you want?) and each student cannot be added more than once to a grade or the roster.
-In fact, when a test attempts to add the same student more than once, your implementation should indicate that this is incorrect.
+Note that all our students only have one name (it’s a small town, what do you want?), and each student cannot be added more than once to a grade or the roster. If a test attempts to add the same student more than once, your implementation should indicate that this is incorrect.
```
Comment (BNAndras) [inline exercises/grade-school/description.md:18]: One sentence per line please.

Comment (Elahi-cs) [reply exercises/grade-school/description.md]: Oops, I completely ignored the original format there and just went with my own style. My bad!

**Comment by IsaacG** [comment]
I'm reluctant to change minor things that aren't actually incorrect. We don't want everyone that wants to contribute to come through and tweak language to their preference. That introduces a lot of churn. Feel free to correct grammatical errors. Please don't word smith just to word smith without prior approval via a forum discussion.

**Comment by Elahi-cs** [comment]
> I'm reluctant to change minor things that aren't actually incorrect. We don't want everyone that wants to contribute to come through and tweak language to their preference. That introduces a lot of churn. Feel free to correct grammatical errors. Please don't word smith just to word smith without prior approval via a forum discussion.

@IsaacG, thank you for your feedback. As stated in the PR description, there has already been a discussion in the forum, which I will link again [here](https://forum.exercism.org/t/improve-grade-school-description/13694).

**Comment by IsaacG** [comment]
> @IsaacG, thank you for your feedback. As stated in the PR description, there has already been a discussion in the forum, which I will link again [here](https://forum.exercism.org/t/improve-grade-school-description/13694).

My apologies.

File: exercises/grade-school/description.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  - "We've only got Jim just now."
+  - "We've only got Jim right now."
 - Get a sorted list of all students in all grades.
-  Grades should sort as 1, 2, 3, etc., and students within a grade should be sorted alphabetically by name.
-  - "Who all is enrolled in school right now?"
+  Grades should be sorted as 1, 2, 3, etc., and students within a grade should be sorted alphabetically by name.
+  - "Who is enrolled in school right now?"
   - "Let me think.
-    We have Anna, Barb, and Charlie in grade 1, Alex, Peter, and Zoe in grade 2 and Jim in grade 5.
-    So the answer is: Anna, Barb, Charlie, Alex, Peter, Zoe and Jim"
```
Comment (IsaacG) [inline exercises/grade-school/description.md:18]: These lines are part of the bullet point and should be indented.

**Review by IsaacG — APPROVED** [review]

**Review by Cool-Katt — APPROVED** [review]

File: exercises/grade-school/description.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+    We have Anna, Barb, and Charlie in grade 1, Alex, Peter, and Zoe in grade 2, and Jim in grade 5.
+    So the answer is: Anna, Barb, Charlie, Alex, Peter, Zoe, and Jim."
 
-Note that all our students only have one name (It's a small town, what do you want?) and each student cannot be added more than once to a grade or the roster.
-In fact, when a test attempts to add the same student more than once, your implementation should indicate that this is incorrect.
+Note that all our students only have one name (it’s a small town, what do you want?), and each student cannot be added more than once to a grade or the roster.
```
Comment (tasxatzial) [inline exercises/grade-school/description.md:20]:
```suggestion
Note that all our students only have one name (it's a small town, what do you want?), and each student cannot be added more than once to a grade or the roster.
```

**Review by tasxatzial — COMMENTED** [review]
Just a small fix for consistency

File: exercises/grade-school/description.md (hunk 3 above)
Comment (Elahi-cs) [reply exercises/grade-school/description.md]: Well spotted, thanks!

**Review by tasxatzial — APPROVED** [review]

**Review by IsaacG — APPROVED** [review]

**Review by keiravillekode — APPROVED** [review]

**Comment by IsaacG** [comment]
+cc @ErikSchierboom

**Pull request merged**